src/Views/Form/Themes/Sequoia/sections/introduction.php

```diff
@@ -1,17 +1,25 @@
 <?php
+global $post;
 
 // Get headline and description
-$headline    = $this->themeOptions['introduction']['headline'];
-$description = $this->themeOptions['introduction']['description'];
-$image       = $this->themeOptions['introduction']['image'];
+$headline    = ! empty( $this->themeOptions['introduction']['headline'] ) ? $this->themeOptions['introduction']['headline'] : $post->post_title;
+$description = ! empty( $this->themeOptions['introduction']['description'] ) ? $this->themeOptions['introduction']['description'] : $post->post_excerpt;
+$image       = ! empty( $this->themeOptions['introduction']['image'] ) ? $this->themeOptions['introduction']['image'] : $post->post_thumbnail;
 ?>
 
 <div class="give-section introduction">
-	<img src="<?php echo $image; ?>" />
-	<div class="heading">
-		<h2 class="give-form-title"><?php echo $headline; ?></h2>
-	</div>
-	<div class="subheading text">
-		<p><?php echo $description; ?></p>
-	</div>
+	<h2>
+		<?php echo $headline; ?>
+	</h2>
+	<?php if ( ! empty( $description ) ) { ?>
```

Comment:
When we have `HTML` content between the PHP tags then we can use `if/endif` to make code looks better.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/574af9f781844d374ad8aa5310e9f2c5f815e012